### PR TITLE
Add intro feature handling and improve argument parsing

### PR DESCRIPTION
**Pull request by Mezzair**
Add intro feature handling and improve argument parsing

This commit introduces the handling of the `--intro` command line argument which controls the activation of the intro functionality in the script. The intro feature is related to managing a 'precount' checkbox on the web interface, which is now conditionally checked based on the presence of this argument.

Changes:
- Parse the `--intro` flag from command line arguments to set the `intro` boolean variable.
- Add conditional logic to check the 'precount' checkbox when the `intro` flag is true.
- Enhance the debug output to include logging for all major variables influenced by command line arguments, aiding in troubleshooting and verification of script behavior.

This enhancement ensures that users can opt into precount features dynamically via the command line, improving the script's flexibility and usability in different use cases.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,12 +11,14 @@ if (!process.env.KV_USERNAME || !process.env.KV_PASSWORD) {
...
-if (args[0] =~ /node$/) {
+if (args[0].match(/node$/)) {
   args = args.slice(2);
-} else if (args[0] =~ /npm$/) {
+} else if (args[0].match(/npm$/)) {
   args = args.slice(3);
 }
 
+console.log(args);
+
```
Comment (subdigital) [inline src/index.js:21]: let's remove the console logs here and below, I'm sure they were helpful in validating this change worked, but it's just noise for end users.

Comment (Mezzair) [reply src/index.js:21]: My bad.

File: src/index.js [hunk 2]
```diff
@@ -28,6 +28,13 @@ if (!songUrl) {
 let downloadPath = util.getArgValue(args, "-d");
 let headless = args.includes("-h") || args.includes("--headless");
 let pitch = parseInt(util.getArgValue(args, "-p"));
+let intro = args.includes("-i") || args.includes("--intro");
+
+console.log(`Download Path: ${downloadPath}`);
+console.log(`Headless Mode: ${headless}`);
+console.log(`Pitch Value: ${pitch}`);
+console.log(`Intro Flag: ${intro}`);
```
[the remark this answers is not in the document]
Comment (Mezzair) [reply src/index.js:36]: I'd argue it helps confirm that you've sent the parameters through correctly?

[the remark this answers is not in the document]
Comment (subdigital) [reply src/index.js:36]: let's remove -- I think the readme is sufficient here

[the remark this answers is not in the document]
Comment (nicola-lunghi) [reply src/index.js:36]: resubmited this PR here https://github.com/subdigital/puppeteer-karaoke-version/pull/6
with the changes requested

**Comment by subdigital** [comment]
Closed in favor of #6

**Pull request closed without merging**